src/EnergyPlus/Construction.hh

```diff
@@ -268,6 +271,7 @@ namespace Construction {
         // with a heat source/sink.
         int NodeSource;   // Node at which a source or sink is present
         int NodeUserTemp; // Node where user wishes to calculate a temperature (for constructions with sources/sinks only)
+        bool CTFLoadedFromCache;
```

Comment:
New construction member variable for checking whether the CTFs have been loaded from the cache.